shared/haml/hamburger.haml

```diff
@@ -70,6 +78,8 @@
       show_teacher_options = "show-mobile"
     elsif user_type == "student"
       show_student_options = "show-mobile"
+    else
+      show_not_signed_in_options = "show-mobile"
```

Comment:
Given these options can now appear, do we now need to add them to the code below that determines if/when we show the hamburger at all?